search/search-domain/src/main/java/io/camunda/search/filter/ProcessInstanceFilter.java

```diff
@@ -39,7 +39,7 @@ public record ProcessInstanceFilter(
     List<Operation<String>> flowNodeIdOperations,
     Boolean hasFlowNodeInstanceIncident,
     List<Operation<String>> flowNodeInstanceStateOperations,
-    List<Integer> incidentErrorHashCodes,
+    Integer incidentErrorHashCode,
```

Comment:
❓ Can we make this a list of operations as well so it's consistent with the other filter criteria?